src/unity/lib/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 project(unity)
 
-subdirs(visualization)
+subdirs(
+	annotations
```

Comment:
Hmm. Maybe we should consolidate `annotations` and `visualization` into something like `ui`?

Reply:
Good Idea. It'll help considerate the UI code